dali/python/nvidia/dali/_multiproc/pool.py

```diff
@@ -163,20 +172,22 @@ def handle_error(self, batch_i):
                 else:
                     # Raise new exception propagating the traceback from worker thread as error
                     # message, originating from original exception
-                    raise Exception(
-                        "\n\nException traceback received from worker thread:\n\n" + traceback_str) from exception
+                    raise Exception("\n\nException traceback received from worker thread:\n\n" +
+                                    traceback_str) from exception
```

Comment:
binary operator on new line is preferred (although not enforced)
```suggestion
                    raise Exception("\n\nException traceback received from worker thread:\n\n"
                                    + traceback_str) from exception
```

Reply:
Done